### PR TITLE
Create App FEB2021 bundle

### PR DESCRIPTION
**Pull request by AleziaKurdis**
This PR contains:

**Menu reorganization**
EntityList menubar has been reorganized, introducing a new menu “**Transform**”
The menu items has been re-dispatched. CSS has been retouched to display clearly which menu is open.

image

**New optional full path & file name URL column in the EntityList**
A new optional column “URL” has been added (hidden by default) to the EntityList. (Issue #977)
It displays the full URL including the path. The column is sortable. It is also searchable, but only if visible.

image

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-fc857d0.tar_6ffb8485.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-fc857d0.tar_6ffb8485.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-fc857d0_6ffb8485.sh

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-fc857d0_6ffb8485.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-fc857d0_6ffb8485.exe

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-Quest-Alpha-PR1030-fc857d0_6ffb8485.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-fc857d0_6ffb8485.apk

**Comment by Aitolda** [comment]
Working.  No complaints.  Just used it very quickly to sort through a domain to make sure all my models were on my S3.  Also transform menu elements are working as expected.

File: scripts/system/create/entityList/html/js/entityList.js [hunk 1]
```diff
@@ -929,13 +943,14 @@ function loaded() {
                 entityData.forEach(function(entity) {
                     let type = entity.type;
                     let filename = getFilename(entity.url);
-            
+
```
Comment (ctrlaltdavid) [inline scripts/system/create/entityList/html/js/entityList.js:946]: Unnecessary whitespace.

File: scripts/system/create/entityList/html/js/entityList.js [hunk 2]
```diff
@@ -975,11 +990,13 @@ function loaded() {
                         let searchFilter = searchTerm === '' || (e.name.toLowerCase().indexOf(searchTerm) > -1 ||
                                                                  e.type.toLowerCase().indexOf(searchTerm) > -1 ||
                                                                  e.fullUrl.toLowerCase().indexOf(searchTerm) > -1 ||
+                                                                 (e.urlWithPath.toLowerCase().indexOf(searchTerm) > -1 && 
+                                                                 columnsByID["urlWithPath"].elTh.style.visibility === "visible")||
```
Comment (ctrlaltdavid) [inline scripts/system/create/entityList/html/js/entityList.js:994]:
```suggestion
                                                                 columnsByID["urlWithPath"].elTh.style.visibility === "visible") ||
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-d150b9c_b2b38ad2.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-d150b9c.tar_b2b38ad2.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-d150b9c_b2b38ad2.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-d150b9c.tar_b2b38ad2.Z

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-d150b9c_b2b38ad2.exe

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-d150b9c_b2b38ad2.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-Quest-Alpha-PR1030-d150b9c_b2b38ad2.apk

**Comment by SilverfishVR** [comment]
testing [Vircadia-PR1030-d150b9c](https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1030/Vircadia-PR1030-d150b9c_b2b38ad2.exel)

tested mostly in VR, works well and all menu's fit in tablet 👍

**Pull request merged**